js/app.js

```diff
@@ -1,10 +1,46 @@
 'use strict';
 
+// number of images displayed at a time. recommended < 10 or time to find a unique set of images gets ridiculous...
```

Comment:
That makes sense! If you have 20 images, and need 11 that are unique from each other and also unique from the last 11 that were displayed... you're gonna have a bad time!